pom.xml

```diff
@@ -22,11 +22,14 @@
     <java.version>1.8</java.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <devon4j.version>3.1.0-SNAPSHOT</devon4j.version>
+    <revision>3.1.0</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <devon4j.version>${revision}${changelist}</devon4j.version>
```

Comment:
in devon-ide I decided to keep the actual version in `.mvn/maven.config` that I added to the git repo. This allows to externalize the version information out of all `pom.xml` files and reduce complexity for release creation (version bumping, setting next SNAPSHOT version). I also set the revision to `dev` in the POM itself to avoid redundancy and to enforce that if the `.mvn/maven.config` got lost, we have an abvious impact and do not accidentially create a wrong release-number (looking like a normal release). I would recommend to do the same here espcially as the redundant values will either lead to maintaining both places in parallel or to cause confusion if they run out of sync.